src/Views/Form/Templates/Sequoia/assets/css/form.scss

```diff
@@ -34,7 +34,7 @@ html {
 
 body {
 	margin: 10px;
-	font-family: 'Montserrat', sans-serif;
+	font-family: Arial, Helvetica, sans-serif;
```

Comment:
Do we want Arial specifically because we think it looks good? Also, Helvetica is a bit pointless here since systems will more likely have Arial and not have Helvetica (macOS has it; Windows/Android doesn’t).

I actually think that we should use `system-ui, sans-serif` instead since that will use the default system font. No support for IE, but sans-serif will be a valid fallback in that case.

What do you think @DevinWalker?